src/controllers/activation.ctrl.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path="../../typings/express-session/express-session.d.ts" />
+import Restify = require('restify');
+import auth = require('../services/user-auth');
+import userService = require('../services/user-service');
+import models = require('../models/models');
+import Promise = require('bluebird');
+
+module ActivationController {
+  export function activate (req:Restify.Request,res:Restify.Response,next){
+    var activate = req.params.activate;
+    //take out after integration tests have passed
+    // var create_user = userService.createUser (
+    //   'bob', 'lino',
+    //   'username2', 'email2@email3@com',
+    //   'phash', 'salt')
+
+
+    userService.activateUser(activate)
+      .then((user) => {
+        //TODO: Activation was successfull. Refirect request to
+        res.header('Location', '/login');
```

Comment:
I just found out you can redirect like this in Restify: `res.redirect('/login', next)`. This way we do not need to mess with the headers.